velodyne_driver/src/driver/driver.cc

```diff
@@ -58,6 +58,11 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
       packet_rate = 754;             // 754 Packets/Second for Last or Strongest mode 1508 for dual (VLP-16 User Manual)
       model_full_name = "VLP-16";
     }
+  else if (config_.model == "VLP32")
```

Comment:
we should call the device `VLP32C`
there will be other velodyne devices that have 32 lasers that will have different density patterns

Reply:
actually, leave this as is